tests/test_scrapers.py

```diff
@@ -21,105 +21,81 @@
-        '/tracker/status/release/oldoldstable',
-        '/tracker/status/release/oldoldstable-backports',
-        '/tracker/status/dtsa-candidates',
-        '/tracker/status/todo',
-        '/tracker/status/undetermined',
-        '/tracker/status/unimportant',
-        '/tracker/status/itp',
-        '/tracker/status/unreported',
-        '/tracker/data/unknown-packages',
-        '/tracker/data/fake-names',
-        '/tracker/data/missing-epochs',
-        '/tracker/data/latently-vulnerable',
-        '/tracker/data/funny-versions',
-        '/tracker/data/releases',
-        '/tracker/data/json',
-    ]
-
-    assert expected == debian.extract_tracker_paths(test_input)
-
-
-def test_debian_extract_cves_from_tracker():
-    test_input = """
-    <tr>
-        <td><a href="/tracker/source-package/389-ds-base">389-ds-base</a></td>
-        <td><a href="/tracker/CVE-2016-5416">CVE-2016-5416</a></td>
-        <td>not yet assigned</td><td>?</td>
-    </tr>
-    """
-
-    expected = (
-        ['CVE-2016-5416'],
-        ['389-ds-base'],
-        ['not yet assigned'],
-    )
-
-    assert expected == debian.extract_cves_from_tracker(test_input)
+    ubuntu_testfile = join(dirname(__file__), 'ubuntu', 'main.html')
+
+    with open(ubuntu_testfile) as f:
+        test_input = f.read()
+
+    cves = ubuntu.extract_cves(test_input)
+
+    expected = {
+        'cve_id': 'CVE-2002-2439',
+        'package_name': 'gcc-4.6',
+        'vulnerability_status': 'low'
+    }
+    assert expected == cves[0]
+
+    expected = {
+        'cve_id': 'CVE-2013-0157',
+        'package_name': 'util-linux',
+        'vulnerability_status': 'low',
+    }
+    assert expected == cves[50]
+
+    expected = {
+        'cve_id': 'CVE-2017-9986',
+        'package_name': 'linux-lts-xenial',
+        'vulnerability_status': 'medium',
+    }
+    assert expected == cves[-1]
+
+
+def test_debian_extract_data():
+    debian_test_file = join(dirname(__file__), 'test_data', 'debian.json')
+
+    with open(debian_test_file) as f:
+        test_input = f.read()
+
+    test_data = json.loads(test_input)
+
+    expected = [{
+                    'fixed_version': '1.50-1.1',
+                    'package_name': 'mimetex',
+                    'status': 'resolved',
+                    'urgency': 'medium',
+                    'vulnerability_id': ''
+                },
+
+                {
+                    'fixed_version': '1.50-1.1',
+                    'package_name': 'mimetex',
+                    'status': 'not-resolved',
+                    'urgency': 'medium',
+                    'vulnerability_id': 'CVE-2009-2459'
+                },
+
+                {
+                    'fixed_version': None,
+                    'package_name': 'git-repair',
+                    'status': 'open',
+                    'urgency': 'unimportant',
+                    'vulnerability_id': 'TEMP-0807341-84E914'
+                },
+
+                {
```

Comment:
Collecting this have no value.
If the 'jessie' data is not available we should not create an entry in the output.